ipv6-in-vnet-StdLB/azuredeploy.json

```diff
@@ -0,0 +1,398 @@
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the administrator of the new VM. Exclusion list: 'admin','administrator'"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password for the administrator account of the new VM"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+	"vmSize": {
+	  "type": "string",
+	  "defaultValue": "Standard_A1",
+	  "metadata": {
+		"description": "Size of VMs to deploy in VNET."
+	  }
+    }
+  },
+  "variables": {
+    "vnetv4AddressRange": "10.0.0.0/16",
+    "vnetv6AddressRange": "ace:cab:deca::/48",
+    "subnetv4AddressRange": "10.0.0.0/24",
+    "subnetv6AddressRange": "ace:cab:deca:deed::/64",
+    "subnetName": "DualStackSubnet",
+    "newStorageAccountName": "[concat('st', uniqueString(resourceGroup().id))]",
+    "availabilitySetName": "myavset",
+    "numberOfInstances": 2,
+    "vmName": "DsVM",
+    "publicipName": "RDPpublicIp",
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "imageSku": "2019-Datacenter"
+  },
+  "resources": [
+    {
+      "apiVersion": "2019-02-01",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "[concat(variables('publicipName'), copyindex())]",
+      "copy": {
+        "name": "publicIpLoop",
+        "count": "[variables('numberOfInstances')]"
+      },
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "Standard"
+      },
+      "properties": {
+        "publicIPAllocationMethod": "Static"
+      }
+    },
+    {
+      "apiVersion": "2019-02-01",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "lbpublicip",
+      "location": "[parameters('location')]",
+      "sku": {
+      		"name": "Standard"
+      },
+      "properties": {
+        "publicIPAllocationMethod": "Static"
+      }
+    },
+    {
+      "apiVersion": "2019-02-01",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "lbpublicip-v6",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "Standard"
+      },
+      "properties": {
+        "publicIPAllocationMethod": "Static",
+        "publicIPAddressVersion": "IPv6"
+      }
+    },
+    {
+      "type": "Microsoft.Compute/availabilitySets",
+      "name": "[variables('availabilitySetName')]",
+      "apiVersion": "2019-03-01",
+      "location": "[parameters('location')]",
+      "properties": {
+        "platformFaultDomainCount": "2",
+        "platformUpdateDomainCount": "5"
+      },
+      "sku": {
+        "name": "Aligned"
+      }
+    },
+    {
+      "apiVersion": "2019-04-01",
+      "type": "Microsoft.Storage/storageAccounts",
```

Comment:
this is never used

Reply:
Brian, please explain...what exactly is never used?

Reply:
The storage account - we're creating one in the template, but it's not used for anything...